.github/workflows/scripts/check_ignore.py

```
@@ -0,0 +1,259 @@
+    print(
+        subprocess.check_output(["git", "branch"]).decode("utf-8").strip("\n")
+    )
+
+    current_branch = (
+        subprocess.check_output(["git", "branch", "--show-current"])
+        .decode("utf-8")
+        .strip("\n")
+    )
+    print(current_branch)
+    changed_files = (
+        subprocess.check_output(
```

Ruff (0.14.11)

`subprocess` call: check for execution of untrusted input

(S603)